src/cmd/serve.rs

```diff
@@ -196,6 +205,10 @@ pub fn serve(interface: &str, port: &str, output_dir: &str, config_file: &str) -
                                 console::info(&format!("-> Sass file changed {}", path.display()));
                                 rebuild_done_handling(&broadcaster, site.compile_sass(&site.base_path), &p);
                             },
+                            (ChangeKind::Config, _) => {
+                                console::info(&format!("-> Config changed. The whole site will be reloaded. The browser needs to be refreshed to make the changes visible."));
+                                site = rebuild::after_config_change(interface, port, output_dir, config_file).unwrap();
```

Comment:
i think you can just call `create_new_site` here without having anything in `rebuild.rs`

Reply:
Indeed!